src/binary_tree.rs

```diff
@@ -551,15 +567,15 @@ mod tests {
         let right_node = single_leaf(x_coord).into_node();
         let sibling = Sibling::from_node(right_node);
         match sibling {
-            Sibling::Left(_) => panic!("Node should be a right sibling"),
+            Sibling::Left(_) => std::panic!("Node should be a right sibling"),
```

Comment:
Why have the extra module specifier? It doesn't change which function is called

Reply:
The compiler was throwing an error. Something to do with ambiguity regarding which `panic!` macro is being used. Specifying the `std` module one sorted it.